src/main/resources/cms-mig.json

```diff
@@ -0,0 +1,128 @@
+{
+  "scehma": "https://json-schema.org/draft/2019-09/schema",
+  "title": "CMS DECS Migration",
```

Comment:
We should also specify a version which the schema is using, such as swagger: "2.0" and those that match openapi: 3.0.n (for example, openapi: 3.0.0).

Also means then you can view the models using an openapi/swagger viewer like https://editor.swagger.io/